### PR TITLE
fix(searchbox): suggestions css

### PR DESCRIPTION
**Pull request by matijakljajic**
Small fixes regarding css.

### CODE REVIEW

File: src/lib/components/searchbox/main.svelte [hunk 1]
```diff
@@ -173,7 +173,7 @@
 			class:border-neutral-200={showSuggestions}
 			class:dark:border-b-2={showSuggestions}
 			class:rounded-full={!showSuggestions}
-			class="size-full flex dark:bg-neutral-800 dark:shadow-none border dark:border-0 border-neutral-100 dark:border-neutral-700"
+			class="size-full flex dark:bg-neutral-800 dark:shadow-none border box-content dark:border-0 border-neutral-100 dark:border-neutral-700"
```
Comment (aleksasiriski) [inline src/lib/components/searchbox/main.svelte:176]: Let's keep all of the border classes next to each other, I know it's a small nitpick but I've refactored the whole project around that so it's easier to understand all TailwindCSS styles when reading the code.

I would put this before size-full i.e. at the start like TW docs [here](https://tailwindcss.com/docs/box-sizing#excluding-borders-and-padding).

File: src/lib/components/searchbox/main.svelte [hunk 2]
```diff
@@ -254,14 +254,14 @@
...
-					class="rounded-lg"
+					class="px-4 py-0.5 rounded-lg"
 				>
 					<button
 						onmouseover={() => (currentIndex = i)}
 						onfocus={() => (currentIndex = i)}
 						onclick={() => (clickedIndex = i)}
 						type="submit"
-						class="px-4 py-0.5 size-full text-left"
+						class="size-full text-left"
```
Comment (aleksasiriski) [inline src/lib/components/searchbox/main.svelte:264]: Was this change necessary? This makes it so you HAVE to click on the suggestion text for it to register the click, whereas before it was purposefuly made so you can click on the whole hovered area (left to the text, under the text, everything that has the lighter color in this img).
image

**Review by aleksasiriski — CHANGES_REQUESTED** [review]

File: src/lib/components/searchbox/main.svelte (hunk 1 above)
Comment (matijakljajic) [reply src/lib/components/searchbox/main.svelte:176]: I've put it there itentionally as it solves the border problem. That was my logic behind the placement as it's directly related to the borders in this instance. I'll reform it as you please though, it's no problem.

File: src/lib/components/searchbox/main.svelte (hunk 2 above)
Comment (matijakljajic) [reply src/lib/components/searchbox/main.svelte]: This was a misunderstanding and an honest mistake, figured it out over Discord. Pesky `class:pb-1={last}` I did not see at first was the problem which weirded me out. I changed it to `class:pb-0.5={last}` since it appears a lot less extreme. Hope that looks good to you as it does to me.

**Review by aleksasiriski — APPROVED** [review]
